Dịch "selecting-servers-gpus_vn" - Phần 1 (#3024)
aivivn/d2l-vn: chapter_appendix-tools-for-deep-learning/selecting-servers-gpus_vn.md

and [G4](https://aws.amazon.com/blogs/aws/in-the-works-ec2-instances-g4-with-nvidia-t4-gpus/) instances are a much more practical solution.
-->

Việc huấn luyện học sâu thông thường đòi hỏi một lượng lớn tính toán.

> Ở thời điểm hiện tại, GPU là công cụ tăng tốc phần cứng chi phí hiệu quả nhất cho việc học sâu.

Reviewer: ```suggestion
Ở thời điểm hiện tại, GPU là công cụ tăng tốc phần cứng có hiệu quả về chi phí nhất cho việc học sâu.
```

Cụ thể, so với CPU, GPU rẻ hơn và cung cấp hiệu suất cao hơn, thường là hơn một bậc.

> Hơn nữa, một máy chủ đơn có thể hỗ trợ đa GPU, tới 8 bộ cho các máy chủ cao cấp.

Reviewer: ```suggestion
Hơn nữa, một máy chủ đơn có thể hỗ trợ đa GPU, tới 8 GPU với các máy chủ cao cấp.
```

> Số GPU điển hình hơn là đến 4 cho một máy trạm kỹ thuật,

Reviewer: ```suggestion
Số GPU điển hình hơn là 4 cho một máy trạm kỹ thuật,
```

> vì vấn đề đòi hỏi về tỏa nhiệt, làm mát và lượng điện tiêu thụ tăng vọt nhanh chóng vượt quá khả năng của một tòa nhà văn phòng có thể cung cấp.

Reviewer: ```suggestion
vì vấn đề tỏa nhiệt, làm mát và lượng điện tiêu thụ sẽ tăng vọt  vượt quá khả năng một tòa nhà văn phòng có thể cung cấp.
```

> Để triển khai trên số lượng lớn hơn, điện toán đám mây, chẳng hạn như [P3] của Amazon (https://aws.amazon.com/ec2/instance-types/p3/)
> và các bản mẫu [G4](https://aws.amazon.com/blogs/aws/in-the-works-ec2-instances-g4-with-nvidia-t4-gpus/) là một giải pháp thực tế hơn nhiều.

Reviewer: ```suggestion
Để triển khai trên số lượng lớn hơn, điện toán đám mây, chẳng hạn như các máy ảo [P3](https://aws.amazon.com/ec2/instance-types/p3/) và [G4](https://aws.amazon.com/blogs/aws/in-the-works-ec2-instances-g4-with-nvidia-t4-gpus/) của Amazon là một giải pháp thực tế hơn nhiều.
```

> 

Reviewer: Đoạn phía dưới chưa dịch anh.

Follow the guidelines below if possible:
-->

> *dịch đoạn phía trên* 

Reviewer 2: @LongNguyen1984 nhờ anh dịch giúp em đoạn này nữa anh nha. Em cảm ơn anh ạ.

Author: Đã bổ sung

This is partly due to the number of PCIe lanes that the CPU offers.
-->

1. **Bộ Nguồn Cấp Điện **. GPU sử dụng một lượng điện năng đáng kể.
Nguồn cung lên đến 350W cho mỗi thiết bị (kiểm tra *công suất đỉnh* của card đồ họa thay vì công suất trung bình,

> vì mã hiệu quả cho việc triển khai có thể sử dụng nhiều năng lượng).

Reviewer: ```suggestion
vì mã nguồn hiệu quả có thể ngốn nhiều năng lượng).
```

Nếu nguồn điện của bạn không đáp ứng được nhu cầu, bạn sẽ thấy rằng hệ thống của mình trở nên không ổn định.

> 1. **Kích thước khung**. GPU có kích thước lớn và các đầu nối nguồn phụ trợ thường cần thêm không gian.

Reviewer: ```suggestion
2. **Kích thước khung chứa**. GPU có kích thước lớn và các đầu nối nguồn phụ trợ thường cần thêm không gian.
```

Thêm nữa, khung máy lớn giúp dễ làm mát hơn.

> 1. **Làm mát GPU**. Nếu bạn có số lượng lớn GPU, bạn có thể muốn đầu tư vào tản nhiệt nước.

Reviewer: ```suggestion
3. **Làm mát GPU**. Nếu bạn có số lượng lớn GPU, bạn có thể muốn đầu tư vào tản nhiệt nước.
```

> Ngoài ra, với mục tiêu làm *thiết kế tham khảo* ngay cả khi chúng có số quạt làm mát ít hơn, vì chúng đủ mỏng để cho phép hút gió giữa các thiết bị.

Reviewer: ```suggestion
Ngoài ra, có thể sử dụng các *thiết kế tham khảo* ngay cả khi chúng có số quạt làm mát ít hơn, vì chúng đủ mỏng để cho phép thông gió giữa các thiết bị.
```

> Nếu bạn mua một GPU có nhiều quạt, nó có thể quá dày để nhận đủ không khí khi lắp đặt nhiều GPU và bạn sẽ gặp phải tình trạng điều tiết nhiệt.

Reviewer: ```suggestion
Nếu bạn mua một GPU có nhiều quạt, nó có thể quá dày để nhận đủ không khí khi lắp đặt nhiều GPU và bạn sẽ gặp phải tình trạng khó tản nhiệt.
```

> 1. **Khe cắm PCIe**. Việc dịich chuyển dữ liệu đến và đi từ GPU (và trao đổi giữa các GPU) đòi hỏi nhiều băng thông.

Reviewer: ```suggestion
4. **Khe cắm PCIe**. Việc dịich chuyển dữ liệu đến và đi từ GPU (và trao đổi giữa các GPU) đòi hỏi nhiều băng thông.
```

Chúng tôi đề xuất khe cắm PCIe 3.0 với 16 làn. Nếu bạn lắp nhiều GPU, hãy đảm bảo là bạn đọc kỹ mô tả bo mạch chủ để chắc chắn 

> băng thông 16x đó vẫn khả dụng khi nhiều GPU được sử dụng cùng lúc và nó có nghĩa là bạn đang nhận được tốc độ PCIe 3.0 thay vì PCIe 2.0 cho các khe cắm bổ sung.

Reviewer: ```suggestion
băng thông 16x đó vẫn khả dụng khi nhiều GPU được sử dụng cùng lúc và tốc độ PCIe là 3.0 thay vì PCIe cho các khe cắm bổ sung.
```

-->

* **Người mới bắt đầu**. Một GPU cấp thấp với mức tiêu thụ điện năng thấp (GPU dành cho chơi game giá rẻ phù hợp cho việc sử dụng học sâu 150-200W).
Nếu bạn may mắn, máy tính hiện tại của bạn sẽ hỗ trợ nó.
* **1  GPU**. Một CPU cấp thấp với 4 nhân sẽ là quá đủ và hầu hết các bo mạch chủ đáp ứng được.
 Hãy nhắm đến DRAM với ít nhất 32 GB và đầu tư vào một ổ SSD để truy cập dữ liệu cục bộ.
 Nên sử dụng nguồn cung cấp 600W là đủ. Mua GPU có nhiều quạt.
* **2 GPU **. Một CPU cấp thấp với 4-6 nhân là đủ. Hãy nhắm đến DRAM 64 GB và đầu tư vào một ổ SSD.
Bạn sẽ cần nguồn tầm 1000W cho hai GPU cao cấp. Đối với bo mạch chủ, hãy đảm bảo rằng chúng có *hai* khe cắm PCIe 3.0 x16.
Nếu có thể, hãy mua một bo mạch chủ có hai khoảng trống (khoảng cách 60mm) giữa các khe PCIe 3.0 x16 để có thêm không khí.
Trong trường hợp này, hãy mua hai GPU có nhiều quạt.
* **4 GPU **. Đảm bảo rằng bạn mua một CPU có tốc độ luồng đơn tương đối nhanh (cụ thể là tần số xung nhịp cao).
Bạn có thể sẽ cần một CPU có số lượng làn PCIe lớn hơn, chẳng hạn như một con AMD Threadripper.
Bạn có thể sẽ cần bo mạch chủ tương đối đắt tiền để có 4 khe cắm PCIe 3.0 x16 vì chúng có thể cần PLX để ghép kênh các làn PCIe.

> Mua GPU có thiết kế chuẩn hẹp và cho không khí đi giữa các GPU.

Reviewer: ```suggestion
Mua GPU có thiết kế sẵn, hẹp và cho không khí đi giữa các GPU.
```

Bạn cần nguồn điện tầm 1600-2000W và ổ cắm trong văn phòng của bạn có thể không hỗ trợ điều đó.
Máy chủ này có thể sẽ tạo ra *tiếng ồn và nóng*. Bạn hẳn là không muốn đặt nó dưới bàn làm việc của bạn.

> Khuyến nghị sử dụng 128 GB DRAM. Mua một ổ SSD (1-2 TB NVMe) để lưu trữ cục bộ và một số ổ cứng để cấu hình RAID cho lưu trữ dữ liệu của bạn.

Reviewer: ```suggestion
Khuyến nghị sử dụng 128 GB DRAM. Mua một ổ SSD (1-2 TB NVMe) để lưu trữ cục bộ và một số ổ cứng theo cấu hình RAID để lưu trữ dữ liệu của bạn.
```

* **8 GPU **. Bạn cần mua khung máy chủ đa GPU chuyên dụng với nhiều nguồn điện dự phòng (chẳng hạn, 2 + 1 cho 1600W với mỗi bộ nguồn).
Điều này sẽ yêu cầu CPU máy chủ có khe cắm kép, 256 GB ECC DRAM, một cạc mạng nhanh (khuyến nghị 10 GBE),

> và bạn sẽ cần kiểm tra liệu máy chủ có hỗ trợ *yếu tố hình dạng vật lý* cho GPU hay không.

Reviewer: ```suggestion
và bạn sẽ cần kiểm tra liệu máy chủ có hỗ trợ *yếu tố vật lý* cho GPU hay không.
```

> Luồng không khí và bố trí đi dây có sự khác biệt đáng kể giữa GPU tiêu dùng và GPU máy chủ  (cụ thể ở đây là RTX 2080 so với Tesla V100).

Reviewer: ```suggestion
Luồng không khí và bố trí đi dây có sự khác biệt đáng kể giữa GPU tiêu dùng và GPU máy chủ (cụ thể ở đây là RTX 2080 so với Tesla V100).
```

Điều này có nghĩa là bạn có thể không lắp đặt được GPU tiêu dùng vào máy chủ do không đủ khoảng trống cho cáp nguồn

> hoặc thiếu dây đai phù hợp (như một trong những đồng tác giả đau đớn phát hiện ra).

Reviewer: ```suggestion
hoặc thiếu dây nối phù hợp (như một trong các đồng tác giả đau đớn phát hiện ra).
```
